src/overlays/actors/ovl_En_Talk_Gibud/z_en_talk_gibud.h

```diff
@@ -3,15 +3,122 @@
 
 #include "global.h"
 
+#define EN_TALK_GIBUD_REQUESTED_ITEM_INDEX(thisx) ((thisx)->params & 0xF)
+#define EN_TALK_GIBUD_SWITCH_FLAG(thisx) (((thisx)->params & 0xFF0) >> 4)
+
+typedef enum {
```

Comment:
Sure, one sec...

Reply:
I'm leaving some enums in the header for now, because they are common to other redeads/gibdos too. When I clean them up, I will probably put the enums that are currently in talk_gibud.h into rd.h, and then `#include` that in the other gibdo files.

Reply:
Yeah, that could be a good way to do it... I don't suppose there's any possible struct-extending going on amongst them?

Reply:
I don't think so, but Railgibud and Rd are not documented, so I'll see when I get to it. Once this PR gets merged, I'll take a look at those other two actors.